openff/evaluator/protocols/utils.py

```diff
@@ -377,71 +396,19 @@ def generate_reweighting_protocols(
     return protocols, data_replicator
 
 
-def generate_equilibration_protocols(
+def generate_conditional_equilibration_protocols(
```

Comment:
This could have been diffed smarter. This essentially moves a solid chunk of code out of `generate_equilibration_protocols` so both `generate_equilibration_protocols` and `generate_preequilibrated_simulation_protocols` can generate conditional equilibration groups.